src/config.rs

```diff
@@ -1,6 +1,12 @@
 use std::iter::{IntoIterator, Iterator};
 use std::slice::Iter;
 use {grammar, Network, ParseError, ScopedIp};
+use std::net::{IpAddr, Ipv4Addr, Ipv6Addr};
+
+use hostname::get_hostname;
```

Comment:
Let's put it under a feature gate. I like that the parser could be platform independent, i.e. one may want to compile it to wasm and put an online validator for resolv.conf file.

Reply:
feature name? `unix`?

Reply:
maybe other methods on config should be put in this feature? like `glibc_normalize`, `get_nameservers_or_local` which both for linux.

Reply:
I believe `hostname` crate works on windows/OS X too. Maybe `os` or `system` ?

Reply:
And things that don't rely on platform API's shouldn't be feature-gated. I.e. there are use cases to validate linux'ish resolv.conf on OS X or Windows (for example to prepare a docker container).

So `glibc_normalize` and `get_nameservers_or_local` should be kept without feature gate.